### PR TITLE
Docs Update 2.6 changelog with Digital Ocean Modules

### PR DESCRIPTION
**Pull request by BondAnthony**
##### SUMMARY

Included new Digital Ocean modules in the stable-2.6 changelog. 

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

changelog
##### ANSIBLE VERSION

```
stable-2.6
```

### CODE REVIEW

File: changelogs/fragments/new-digitalocean-modules.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+new_features:
+- 'New Digital Ocean Cloud module: digital_ocean_account_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_certificate_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_domain_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_floating_ip_facts'
```
Comment (Akasurde) [inline changelogs/fragments/new-digitalocean-modules.yml]: This is added in version 2.5

File: changelogs/fragments/new-digitalocean-modules.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- 'New Digital Ocean Cloud module: digital_ocean_image_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_load_balancer_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_region_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_size_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_snapshot_facts'
+- 'New Digital Ocean Cloud module: digital_ocean_sshkey_facts'
```
Comment (Akasurde) [inline changelogs/fragments/new-digitalocean-modules.yml]: This is added in version 2.5.

File: changelogs/fragments/new-digitalocean-modules.yml (hunk 1 above)
Comment (BondAnthony) [reply changelogs/fragments/new-digitalocean-modules.yml]: Oops, I’ll update

File: changelogs/fragments/new-digitalocean-modules.yml (hunk 2 above)
Comment (BondAnthony) [reply changelogs/fragments/new-digitalocean-modules.yml]: Oops, I’ll update

**Comment by BondAnthony** [comment]
@Akasurde please take a look now.

**Comment by mattclay** [comment]
New module entries for the changelog will be automatically generated.

**Pull request closed without merging**